File: cc/tests/fixtures/hello/hello_test.cc
// M22 seed C++ test; consumer of cc_test.
#include <cassert>

#include "cc/tests/fixtures/hello/hello.h"

int main() {
  assert(Add(2, 3) == 5);
  assert(Add(-1, 1) == 0);
  return 0;
}

File: ruby/tests/fixtures/hello/hello_test.rb
# Seed Ruby test; consumer of ruby_test (plain executable: exit
# code is the verdict; the RSpec runner mapping is qualified separately
# in `ruby/tests/fixtures/rspec/`).
require_relative "hello"

got = Hello.hello("world")
if got != "hello world"
  warn "FAIL: got '#{got}'"
  exit 1
end
puts "PASS"
